gems/smithy/spec/fixtures/shapes/model.json

```diff
@@ -1,6 +1,21 @@
 {
     "smithy": "2.0",
     "shapes": {
+        "smithy.ruby.tests#BigDecimal": {
```

Comment:
Consider using the AllShapes fixture for your test?

Reply:
I did consider that but noticed that simple shapes defined within AllShapes fixture will reference prelude shapes. However, I'm open to updating to use AllShapes fixture (if we'd like that, I can include that in my next change)